Tests/Tests.Kerberos.NET/Tests.Kerberos.NET.csproj

```diff
@@ -7,6 +7,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
+    <DefineConstants>$(DefineConstants);WEAKCRYPTO</DefineConstants>
```

Comment:
Is this OK?

Just thinking loud:
Maybe we should add an explicit test-target for .NET Full too.
And let CI run on Linux and MacOS too.
If so, then we should file an issue for this.